packages/react-components/src/components/chart/types.ts

```diff
@@ -89,4 +92,52 @@ export type ChartOptions = {
   significantDigits?: number;
   graphic?: InternalGraphicComponentGroupOption[];
   theme?: string;
+  groupId?: string;
+  id?: string;
 };
+
+export interface TrendCursorProps {
+  graphic: InternalGraphicComponentGroupOption[];
+  size: SizeConfig;
+  setGraphic: Dispatch<SetStateAction<InternalGraphicComponentGroupOption[]>>;
+  viewport?: Viewport;
+  series: SeriesOption[];
+  yMax: number;
+  yMin: number;
```

Comment:
Why are `yMax` and `yMin` needed and required?

Reply:
for the TC markers i.e the markers of the TC and series lines intersection. we can use the X of the click location but there is no way to know the Y of the intersection. so to calculate this, need the painted y axis min and max, so the series value could be interpolated